sdk/highlight-go/gorm.go

```diff
@@ -0,0 +1,169 @@
+package highlight
```

Comment:
@Vadman97 is this where you would put this code? Thought about putting it into a separate package, similar to what we did with `hlog`.

Reply:
having just discussed it, i feel like having everything in the same `highlight` package is an easier experience for customers to set up? i think this is fine and i can take an action item to move our logrus instrumentation into the same.